cms/bundles/wagtail_hooks.py

```diff
@@ -38,21 +37,19 @@ def register_viewset() -> list:
     return [bundle_viewset, bundle_chooser_viewset, bundle_page_chooser_viewset]
 
 
-class PageAddToBundleButton(PageListingButton):
```

Comment:
https://docs.wagtail.org/en/latest/releases/7.1.html#deprecation-of-pagelistingbutton-snippetlistingbutton-and-userlistingbutton